docs/osf-trivia-list/dasharo-pro-package.md

```diff
@@ -10,38 +10,27 @@ you can download them:
    subscription. Look for an email from Dasharo that includes your subscription
    data. This email will contain several vital pieces of information:
 
-    - **Logs Key:** A key you might need for other operations or support.
```

Comment:
@PLangowski lgos key is used to upload HCLs/backups to the cloud and to match it with the given subscription. I don;t think we can remove it that easily. Is there any plan to change how the logs/HCLs will be uploaded via DTS?

Reply:
HCL migrated to MinIO according to @PLangowski